src/config/index.ts

```diff
@@ -49,7 +49,6 @@ export const FEATURE_FLAG_ENABLE_TRUSTS_WEB = getEnvironmentValue('FEATURE_FLAG_
 export const FEATURE_FLAG_ENABLE_ROE_UPDATE = getEnvironmentValue('FEATURE_FLAG_ENABLE_ROE_UPDATE_24112022', 'false');
 export const FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME = getEnvironmentValue('FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME_07032023', 'false');
 export const FEATURE_FLAG_ENABLE_UPDATE_TRUSTS = getEnvironmentValue('FEATURE_FLAG_ENABLE_UPDATE_TRUSTS_30062023', 'false');
-export const FEATURE_FLAG_DISABLE_UPDATE_PRIVATE_DATA_FETCH = getEnvironmentValue('FEATURE_FLAG_DISABLE_UPDATE_PRIVATE_DATA_FETCH_28062023', 'false');
```

Comment:
These feature flags used to be in the Terraform config within this project but I see they're now gone. They must be present or set somewhere, so we need to remember to write release note instructions to remove FEATURE_FLAG_DISABLE_UPDATE_PRIVATE_DATA_FETCH.

Reply:
Or does this [PR](https://github.com/companieshouse/ecs-service-configs-dev/pull/487) take care of that?

Reply:
Yes I think so, the feature flags are no longer present within our project and are in [ecs-service-configs-dev](https://github.com/companieshouse/ecs-service-configs-dev/blob/0d35b2a3d1a7592861b9603f27616720c13735bc/development-eu-west-2/cidev/overseas-entities-web.env#L57) (cidev & phoenix1) and [ecs-service-configs-production](https://github.com/companieshouse/ecs-service-configs-production) (staging and live)

> Or does this [PR](https://github.com/companieshouse/ecs-service-configs-dev/pull/487) take care of that?